cmake/modules/FindZSTD.cmake

```diff
@@ -58,6 +58,6 @@ if(ZSTD_FOUND)
     add_library(ZSTD::ZSTD UNKNOWN IMPORTED)
     set_target_properties(ZSTD::ZSTD PROPERTIES
       IMPORTED_LOCATION "${ZSTD_LIBRARIES}"
-      INTERFACE_INCLUDE_DIRECTORIES "${ZSTD_INCLUDE_DIRS}")
+      INTERFACE_INCLUDE_DIRECTORIES "${ZSTD_INCLUDE_DIR}")
```

Comment:
This change makes no difference whatsoever, since `ZSTD_INCLUDE_DIRS` was defined to be `ZSTD_INCLUDE_DIR` to begin with a few lines above:
```cmake
set(ZSTD_INCLUDE_DIRS "${ZSTD_INCLUDE_DIR}")
```
I don't think adding a commit that changes nothing is work it. I'll close the PR, also because @flagarde was not answering to the question of @bellenot.

Feel free to reopen this PR or create a new one if you want to contribute and we were misunderstanding something :slightly_smiling_face::+1: